base/pkg/reqs.jl

```diff
@@ -121,6 +121,11 @@ function dependents(packagename::AbstractString)
     pkgs
 end
 
+function dependencies(packagename::AbstractString)
+  cd(Pkg.dir()) do
```

Comment:
Julia's [CONTRIBUTING.md](https://github.com/JuliaLang/julia/blob/master/CONTRIBUTING.md) asks for 4-space indents.